commit 4b308597838f0fbfdcc3d11845c3065ba61eb57b
Author: Serhei Makarov <smakarov@redhat.com>
Date:   Tue Apr 10 16:13:36 2018 -0400

    RHBZ1558350: fix typo in linux/nfsd.stp
    
    * linux/nfsd.stp (probe nfsd.proc2.create): fix typo.

diff --git a/tapset/linux/nfsd.stp b/tapset/linux/nfsd.stp
index 5a70e84..74124df 100644
--- a/tapset/linux/nfsd.stp
+++ b/tapset/linux/nfsd.stp
@@ -901,7 +901,7 @@ probe nfsd.proc2.create = kernel.function("nfsd_proc_create") !,
 
 	if (@defined($rqstp->rq_argp)) {
 		fh = & @nfsd2_createargs($rqstp->rq_argp)->fh
-		filelen = @nfsd2_createargs($rqstp->rq_argp->len)
+		filelen = @nfsd2_createargs($rqstp->rq_argp)->len
 		filename = kernel_string_n(@nfsd2_createargs($rqstp->rq_argp)->name,
 					   @nfsd2_createargs($rqstp->rq_argp)->len)
 	}
